### DISCUSSION TITLE
Filter-out pdflatex index entries

### DISCUSSION THREAD

**Discussion by mwtoews** [michaeljones/breathe#867] [General]
A PDF version of Sphinx documentation I have been working with from https://proj.org have many of entries in the index section with overprinting errors, e.g.:
![20221106T081626](https://user-images.githubusercontent.com/1230959/200175995-9d4d4065-45b8-4441-a7b9-10026253afd4.jpg)

One simple solution to create better output is to change the default index layout from a 2-column layout to a 1-column layout using sphinx's [LaTeX customization](https://www.sphinx-doc.org/en/master/latex.html). Or removing the index altogether (untested, try setting `'printindex': ''`).

Is it possible to filter-out pdflatex index entries? If so, how?

**Comment by mwtoews** [comment]
Here is the 1-column solution. Modify `conf.py` with something like:
```python
latex_elements['makeindex'] = r'\usepackage[columns=1]{idxlayout}\makeindex'
```
however, it does not "fix" all cases, as there still may be overflow errors like this:
![image](https://user-images.githubusercontent.com/895458/202930305-6053a82b-7f7f-417f-b166-4038291b7ff3.png)

**Comment by michaeljones** [comment]
I'm sorry to say I have no experience with Latex at all and so I'm not in a position to help. Thank you for sharing the issue though and the approach that you have found.
